docs/src/main/asciidoc/deploying-to-openshift-native-howto.adoc

```diff
@@ -0,0 +1,118 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="deploying-to-openshift-native-howto"]
+= Deploy {project-name} applications compiled to native executables
+
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud, native
+:summary: This guide describes how to deploy a Quarkus application to {openshift} compiled to native executables.
+:topics: devops,kubernetes,openshift,cloud,deployment
+:extensions: io.quarkus:quarkus-openshift
+
+You can deploy your native {project-name} applications to {openshift} compiled to native executables by using the Docker build strategy.
+
+You must create a native executable for your application that targets the Linux AMD64 operating system.
```

Comment:
Linux AMD64 feels to restrictive, we test Aarch64 native builds deployed to Aarch64 OCP clusters

Reply:
Addressed with Thomas and removing specific system reference and keeping statement generic.